specs/unified_kernel_image.md

```diff
@@ -159,3 +159,22 @@ that tools staging or consuming UKIs have a common place to store and look for t
 
 The installed UKIs should have a filename `<version format specification>.efi`, i.e. the filename is left to
 implementers but must be valid for comparisons according to the [Version Format Specification](version_format_specification.md).
+
+UKI addons (cmdline PE addons, systemd-sysext extensions for the initrd) built centrally by distributions and
+installed via package manager should be installed into locations depending on whether they are global UKI addons,
+that should be applied to all UKIs installed in the ESP, or whether they should only be applied to a
+single specific UKI.
+
+Global UKI addons should be installed into `/usr/lib/modules/uki.extra.d/`. UKI-specific addons should be installed
```

Comment:
I'd not use the word "Global" here, since it might not be clear what it means.

hence maybe:

"UKI addons that shall apply to *all* installed kernels may be installed into `/usr/lib/modules/uki.extra.d/`. UKI addons that shall apply to a specific installed kernel may be installed into  `/usr/lib/modules/$UNAME/$UKI.efi.extra.d/`, where …"